js/demo-apps/packages/react-front-end/cypress/plugins/index.js

```diff
@@ -36,7 +36,39 @@ module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
     const options = {
-        printLogsToConsole: process.env.VERBOSE_CYPRESS_LOGS || 'onFail'
+        printLogsToConsole: 'always'
     }
     require('cypress-terminal-report/src/installLogsPrinter')(on, options);
-}
+
+    on('before:browser:launch', (browser = {}, launchOptions) => {
+        // if (browser.family === 'chromium' && browser.name !== 'electron') {
+            // auto open devtools
+        launchOptions.args.push('--auto-open-devtools-for-tabs');
+        launchOptions.args.push('--unsafely-treat-insecure-origin-as-secure=http://tokenizer-backend:37766,http://application-back-end:3001,http://application-back-end:3002');
```

Comment:
oh boy haha you went deep

Reply:
This hurt. We spent a long time figuring out WTF was going on. Painful!